### PR TITLE
add bnbchain icon for opBNB

### PR DESCRIPTION
**Pull request by huihzhao**

### CODE REVIEW

**Review by blackdoc111 — APPROVED** [review]

**Comment by huihzhao** [comment]
@blackdoc111 if you could help me to identify the problem, it would be great.

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

I could not resolve your CID with `ipfs get` in a reasonable time - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.

**Comment by huihzhao** [comment]
Could you help review the latest commits? thanks

**Comment by huihzhao** [comment]
> The file you put in iconDownload ends with .png - this is no valid CID

sorry, have corrected the file name

File: _data/iconsDownload/bafybeib75gwytvblyvjpfminitr3i6mpat3a624udfsqsl5nysf5vuuvie
Comment (huihzhao) [inline _data/iconsDownload/bafybeib75gwytvblyvjpfminitr3i6mpat3a624udfsqsl5nysf5vuuvie:1]: renamed the file

**Review by huihzhao — COMMENTED** [review]
updated the icon file

**Review by ligi — APPROVED** [review]

**Pull request merged**